src/test/java/org/ilgcc/app/journeys/GccFlowJourneyTest.java

```diff
@@ -524,7 +524,7 @@ void fullGccFlow() throws IOException {
     private void verifyPDF() throws IOException {
         File pdfFile = getDownloadedPDF();
 
-//         regenerateExpectedPDF(pdfFile); // uncomment and run test to regenerate the test pdf
+         regenerateExpectedPDF(pdfFile); // uncomment and run test to regenerate the test pdf
```

Comment:
@enyia21 - needs to be commented bad
```suggestion
         // regenerateExpectedPDF(pdfFile); // uncomment and run test to regenerate the test pdf
```